src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0,   maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="stylesheet" href="./images/star-active.svg">
+    <link rel="stylesheet" href="./images/star.svg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="content">
+    <div class="card-content" data-qa="card">
+      <img 
+        class="card-content__image"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+      <div class="card-content__produkt">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card-content__code">
+        Product code: 195434
+      </div>
+      <div class="card-content__reviews">
```

Comment:
I would add more space here for better readability
```suggestion
    <div class="card-content" data-qa="card">
      <img 
        class="card-content__image"
        src="images/imac.jpeg"
        alt="iMac"
      >
      
      <div class="card-content__produkt">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </div>
      
      <div class="card-content__code">
        Product code: 195434
      </div>
      
      <div class="card-content__reviews">
```